README.md

```diff
@@ -179,34 +219,197 @@ Notes and troubleshooting
 
 ### Application Flavors
 
-There are 2 flavors (build variants) of PSLab Android application.
+#### Verify Flutter Installation
+```bash
+flutter doctor
+flutter --version
+```
 
-1. #### Play Store Flavor
-  - Play Store flavor uses Google Maps to display location stored in logs in Data logger.
-2. #### Fdroid Flavors
-  - Fdroid flavor uses Open Street Maps to display location stored in logs in Data logger.
+### Android Development Setup
 
-### Development Setup
+**Android Studio is optional** - you can use command-line tools instead.
 
-Before you begin, you should already have the Android Studio SDK downloaded and set up correctly. You can find a guide on how to do this here: [Setting up Android Studio](http://developer.android.com/sdk/installing/index.html?pkg=studio)
+#### Option 1: Android Studio (Recommended)
+1. Install [Android Studio](https://developer.android.com/studio)
+2. Install Android SDK and accept licenses:
+```bash
+flutter doctor --android-licenses
+```
+
+#### Option 2: Command Line Tools Only
+```bash
+# Install Android SDK command-line tools
+# Set environment variables
+export ANDROID_HOME=$HOME/Android/Sdk
+export PATH=$PATH:$ANDROID_HOME/tools:$ANDROID_HOME/platform-tools
+```
 
-### Setting up the Android Project
-For setting up the PSLab Android project you may follow any of the two methods listed below, that is, you may download the repository zip file or you may directly clone the repository to Android Studio.
+**Android Requirements:**
+- Compile SDK: Latest (managed by Flutter)
+- Min SDK: Latest Flutter default
+- Target SDK: Latest Flutter default
+- Java 17 compatibility (configured in build.gradle)
```

Comment:
Add the versions from the build source. Newer versions may not compile as it is.